Add pass to convert split to many slice

Erik-Lundell opened the pull request:
Change-Id: I15f8a37fe88ba6c7d1e357c9ab897069c3c912b2

Erik-Lundell committed:
Add pass to convert split to many slice

Signed-off-by: Erik Lundell <erik.lundell@arm.com>
Change-Id: I15f8a37fe88ba6c7d1e357c9ab897069c3c912b2

digantdesai commented on backends/arm/passes/convert_split_to_slice.py:
```diff
@@ -0,0 +1,72 @@
+from torch.fx.passes.utils.source_matcher_utils import get_source_partitions
+
+
+class ConvertSplitToSlicePass(ExportPass):
```
Nice, like it how simple you made it. :)

digantdesai commented:
Seems like I messed up the merge conflict resolution. Let me fix the missing `)`.

digantdesai commented:
Actually can you please rebase (force) since there are more conflicts. Thanks.

Erik-Lundell commented:
> Actually can you please rebase (force) since there are more conflicts. Thanks.

Done :)

Erik-Lundell committed:
Use node target rather than get_source_partition

This is a better way to find all aten split nodes,
since get_source_partion misses split nodes added
during lowering.

Signed-off-by: Erik Lundell <erik.lundell@arm.com>
Change-Id: Iff850850341240d22e1012511de23b3efdabf1bb
